ibis/backends/base_sqlalchemy/alchemy.py

```diff
@@ -13,18 +13,23 @@
 from sqlalchemy.engine.interfaces import Dialect as SQLAlchemyDialect
 
 import ibis
+import ibis.backends.base_sqlalchemy.compiler as comp
+import ibis.backends.base_sqlalchemy.transforms as transforms
 import ibis.common.exceptions as com
 import ibis.expr.analysis as L
 import ibis.expr.datatypes as dt
 import ibis.expr.operations as ops
 import ibis.expr.schema as sch
 import ibis.expr.types as ir
 import ibis.expr.window as W
-import ibis.sql.compiler as comp
-import ibis.sql.transforms as transforms
 import ibis.util as util
+from ibis.backends.base_sqlalchemy.compiler import (
```

Comment:
Same. Would be nice if anything inside `base_sqlalchemy` importing something in `base_sqlalchemy` is done with relative paths. So, if we eventually decide to move or rename the module, we need to update the imports from other backends, and not from itself.